### PR TITLE
Added function to update an existing job configuration

### PR DESCRIPTION
**Pull request by danhooper**
I added a function to update a job configuration modeled after the copy_job function.

### CODE REVIEW

**Comment by jansepar** [comment]
@danhooper thanks for the PR, and sorry for the lengthy delay! It would be great to have this added to the README before merging :)

**Comment by danhooper** [comment]
@jansepar Done :)

File: lib/main.js [hunk 1]
```diff
@@ -212,6 +212,29 @@ var init = exports.init = function(host, options) {
...
             );
         },
+        update_job: function(jobname, modifyfunction, callback) {
+            /*
+            Copies a job and allows you to pass in a function to modify the configuration
+            of the job you would like to copy
+            */
+
+            var self = this;
+            self.get_config_xml(jobname, function(error, data) {
```
Comment (jansepar) [inline lib/main.js]: It isn't necessary to do `self = this` here.

**Comment by jansepar** [comment]
There is one small issue, but I'll just merge and fix it myself. Cheers and thanks for the PR!

**Pull request merged**